tutorcodejail/templates/codejail/apps/settings/tutor.py

```diff
@@ -0,0 +1,41 @@
+from .base import *
+
+from codejail.django_integration_utils import apply_django_settings
+
+SECRET_KEY = "{{ CODEJAIL_SECRET_KEY }}"
+ALLOWED_HOSTS = [
+    "{{ CODEJAIL_HOST }}",
+]
+
+#################### Python sandbox ############################################
+
+CODE_JAIL = {
+    'python_bin': '/sandbox/venv/bin/python',
+    # User to run as in the sandbox.
+    'user': 'sandbox',
+
+    # Configurable limits.
+    # Setting all of them to 0 to disable limits in conatiners.
+    'limits': {
+        #
+    	'NPROC': 0,
+        # How many CPU seconds can jailed code use?
+        'CPU': 0,
+        # Limit the memory of the jailed process to something high but not
+        # infinite (512MiB in bytes)
+        'VMEM': 0,
+        # Time in seconds that the jailed process has to run.
+        'REALTIME': 0,
+        # Needs to be non-zero so that jailed code can use it as their temp directory.(10MiB in bytes)
+        'FSIZE': 10485760,
+        # Disable usage of proxy (force thread-safe)
+        'PROXY': 0,
+    },
+
+    # Overrides to default configurable 'limits' (above).
+    # Keys should be course run ids.
+    # Values should be dictionaries that look like 'limits'.
+    "limit_overrides": {},
+}
+
+apply_django_settings(CODE_JAIL)
```

Comment:
Unless I'm mistaken, creating a Django project would require defining a database. A full Django project seems overkill for what you are trying to achieve here, meaning a stateless python interpreter with a very basic API. Can I suggest relying on a more lightweight framework instead, such as Flask?

Reply:
@regisb This is the content of that file in my host:
```  
  /usr/lib{,32,64}/python{2.[4-7],3.[0-9]}/**.{pyc,so}           mr,
  /usr/lib{,32,64}/python{2.[4-7],3.[0-9]}/**.{egg,py,pth}       r,
  /usr/lib{,32,64}/python{2.[4-7],3.[0-9]}/{site,dist}-packages/ r,
  /usr/lib{,32,64}/python3.[0-9]/lib-dynload/*.so            mr,

  /usr/local/lib{,32,64}/python{2.[4-7],3,3.[0-9]}/**.{pyc,so}           mr,
  /usr/local/lib{,32,64}/python{2.[4-7],3,3.[0-9]}/**.{egg,py,pth}       r,
  /usr/local/lib{,32,64}/python{2.[4-7],3,3.[0-9]}/{site,dist}-packages/ r,
  /usr/local/lib{,32,64}/python3.[0-9]/lib-dynload/*.so            mr,

  # Site-wide configuration
  /etc/python{2.[4-7],3.[0-9]}/** r,

  # shared python paths
  /usr/share/{pyshared,pycentral,python-support}/**      r,
  /{var,usr}/lib/{pyshared,pycentral,python-support}/**  r,
  /usr/lib/{pyshared,pycentral,python-support}/**.so     mr,
  /var/lib/{pyshared,pycentral,python-support}/**.pyc    mr,
  /usr/lib/python3/dist-packages/**.so          mr,

  # wx paths
  /usr/lib/wx/python/*.pth r,

  # python build configuration and headers
  /usr/include/python{2.[4-7],3.[0-9]}*/pyconfig.h r,
```
I checked those paths in the codejailservice container and any of them actually exists, I think it is because I am using pyenv. Can we just then get rid of this ?